PF1/Assignment1/TestAssignment.java

```diff
@@ -0,0 +1,37 @@
+package assignment1;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+
+
+public class TestAssignment {
+	OctalConverter converter;
+	@Before
+	public void setup()
+	{
+		converter=new OctalConverter();
+	}
+	
+	@Test
+	public void tGeneral(){
+		
+		int octnum=converter.binaryToOctal(Integer.parseInt("000100"));
+		Assert.assertEquals(04,octnum);
+		}
+	
+	@Test
+	public void tNull()
+		{
+			int octnum=converter.binaryToOctal(Integer.parseInt(""));
+			Assert.assertEquals(null,octnum);
+		}
+	@Test
+	public void tInvalid()
+	{
+		int octnum=converter.binaryToOctal(Integer.parseInt("1234"));
```

Comment:
This test case will always fail as condition check in if is having OR. That condition should have AND to check for binary number.